conf/lookup/BCD-Port.csv

```diff
@@ -11601,7 +11600,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 33286,UK,3,TRA,,Tranmere Seaport - TRA,Tranmere Seaport - TRA,,1,NULL,1,
 33292,UK,2,TSO,,Tresco Airport - TSO,Tresco Airport - TSO,,1,NULL,1,
 33297,UK,3,TRN,,Troon Seapport - TRN,Troon Seapport - TRN,,1,NULL,1,
-33310,UK,3,TYN,,Tyne Seaport - TYN,Tyne Seaport - TYN,,1,NULL,1,
+33310,UK,3,TYN,Newcastle North Shields South Shields,Tyne Seaport - TYN,Tyne Seaport - TYN,,1,NULL,1,
 33311,UK,3,TYM,,Tynemouth Seaport - TYM,Tynemouth Seaport - TYM,,1,NULL,1,

```

Comment:
This doesn't exist, make part of 33310,UK,3,TYN,Newcastle North Shields South Shields,Tyne Seaport - TYN,Tyne Seaport - TYN,,1,NULL,1,